src/vorta/utils.py

```diff
@@ -171,8 +177,10 @@ def choose_file_dialog(parent, title, want_folder=True):
     dialog.setFileMode(QFileDialog.FileMode.Directory if want_folder else QFileDialog.FileMode.ExistingFiles)
     dialog.setParent(parent, QtCore.Qt.WindowType.Sheet)
     if want_folder:
-        dialog.setOption(QFileDialog.Option.ShowDirsOnly)
-    return dialog
+        list_view = dialog.findChild(QTreeView)
+        list_view.setSelectionMode(QAbstractItemView.SelectionMode.MultiSelection)
+    dialog.exec()
+    return dialog.selectedFiles()
```

Comment:
Why do you change that? Using the non-blocking `open `with a slot is probably better then using `exec`.

Reply:
> Why do you change that? Using the non-blocking `open `with a slot is probably better then using `exec`.

For some reason that way was not working out, the directories (if selected multiple) were not getting added to the sources.